rpi3/firmware/rpi3-bplus/COPYING.linux

```diff
@@ -0,0 +1,356 @@
+
```

Comment:
I don't want to get GPL-v2 files in this git, which even more justifies creating another manifest to just clone the firmware (as done already today), instead of copy files like this.